### ISSUE TITLE
Right password, but ERROR: Login failed: Wallet file or password is wrong.

### ISSUE THREAD

**Issue by onionmon** [rippex/ripple-client-desktop#23]
Hi, I'm having trouble retrieving my wallet.

Downloaded Ripple Desktop Wallet and setup an account with a password. I wrote the password down but not the secret key. I deposited quite a few XRP in. I logged out and assumed that the Wallet would just let you back in with the password. I opened the wallet file (has the windows time stamp of when I created the account) and typed in my password. I got the error: Login failed: Wallet file or password is wrong.

I moved the wallet file around into different directories but that didn't work.

I can see my balance in Read Only Mode.

Any tips for getting into my account? Is this a bug with the Desktop Wallet?

Alternatively, can I get any information out from opening the wallet file in notepad? I know my password, the Desktop Wallet seems to be not registering it correctly.

 

I created a new wallet, typed in a password, skipped the secret key, logged out, and tried opening up that same wallet with the password. Same error!: Login failed: Wallet file or password is wrong.

**Comment by onionmon** [comment]
I also tried putting the wallet file and putting the ripple_data directory + the wallet file inside into the directories where I originally saved it (desktop) and in C:\Program Files\Ripple Desktop Wallet\

**Comment by onionmon** [comment]
Using v. 1.4.1. Win 7 Pro 64-bit

**Comment by onionmon** [comment]
Here are two screen capture videos I made with creating a wallet with password qwerty11, with one wallet that works and another wallet that doesn't work. I filmed these videos one after another.
Works: https://vimeo.com/222989021 Doesn't work: https://vimeo.com/222989026

**Comment by TwanKr** [comment]
I have the same problem. Tried to open an existing account, filled in my password and it says - Login failed: Wallet file or password is wrong. Both are 100% right so I have no idea what to do now.

**Comment by markob17** [comment]
Did you guys ever get access to your wallets?

**Comment by TwanKr** [comment]
Yes. I opened within my Ripple Desktop Wallet a new account. The money was now accessible to me.

**Comment by TMENKL** [comment]
I have the exact same problem. Is there already a solution?

**Comment by markob17** [comment]
TMENKL The solution is to use your secret key to create a new wallet. Without secret key pretty much screwed I believe.

**Comment by marsface** [comment]
Hi, it seems to be a bug that corrupts the wallet file when saved in the default locations. Please choose a different location to save your files and you will be fine. We may not fix this since we are working on another wallet.

**Comment by emibitcoin** [comment]
How can I recover my coins?

**Comment by 67Table** [comment]
I also have a Ripple Wallet that I cannot open using my password.  I have also saved it to other locations as suggested above and I still could not get access to my wallet, still had an error message of wallet file or password is wrong.  

I do have the private key saved, how can I use that to get access to my wallet?

**Comment by markob17** [comment]
67Table Simply click "Create new account" when you first launch Ripple wallet and then click on "create a new account with secret key." That's it pretty much.

**Comment by 67Table** [comment]
Thanks this worked very well.

**Comment by emibitcoin** [comment]
I lost the private key. I just remember my username and password. Can you help me?

**Comment by markob17** [comment]
If you don't have private key I believe you can only access your wallet if you still have the wallet file on your computer. If so, simply point rippex to that file and open it. If this is lost, I believe there is no way to retrieve your funds unfortunately. If you do access your wallet, go to rippex settings and re-write down your private key immediately and save to a safe location such as a safe deposit box at a bank. Good luck.

**Comment by denrusso** [comment]
This is happen in case of admin password of mac os, try create new wallet to desktop, you will see that file is create there, than you can move file where you want. it work for me.

**Comment by Fraciscv** [comment]
also, I think the wallet is saved as a hidden file.. so make sure to show hidden files if you are using a mac.  You can enter this into the terminal and voila.

defaults write com.apple.finder AppleShowAllFiles YES

**Comment by denrusso** [comment]
Already checked, but nothing

**Comment by emibitcoin** [comment]
@markob17 
I have the wallet file.
But I do not know how to open it. (I know file location)
Can you help me?

**Comment by markob17** [comment]
emibitcoin This is from my previous comments above, the instructions I believe were there. I modified them below with just the steps.

Launch Rippex, click the big button below create new account to browse for and choose your rippex wallet file. If/when you do access your wallet, go to rippex settings and re-write down your private key immediately and save to a safe location such as a safe deposit box at a bank. If in the future you don't have this key and you also lose your wallet file, you're basically screwed.

**Comment by emibitcoin** [comment]
@markob17 can you help me? 
I can pay if you help me
